### ISSUE TITLE
Problem running the hello world code

### ISSUE THREAD

**Issue by cwbrandsdal** [ExploratoryEngineering/nrf9160-telenor#6]
I have a problem when I flash the hello_world code to the DK.
It just hangs on booting the OS. Yes, I am VERY new to this.
Below is the complete output from console:

```
***** Booting Zephyr OS build v2.0.99-ncs1 *****
Flash region..Domain..Permissions
00 0x00000 0x08000 .Secure..rwxl
01 0x08000 0x10000 .Non-Secure.rwxl
02 0x10000 0x18000 .Non-Secure.rwxl
03 0x18000 0x20000 .Non-Secure.rwxl
04 0x20000 0x28000 .Non-Secure.rwxl
05 0x28000 0x30000 .Non-Secure.rwxl
06 0x30000 0x38000 .Non-Secure.rwxl
07 0x38000 0x40000 .Non-Secure.rwxl
08 0x40000 0x48000 .Non-Secure.rwxl
09 0x48000 0x50000 .Non-Secure.rwxl
10 0x50000 0x58000 .Non-Secure.rwxl
11 0x58000 0x60000 .Non-Secure.rwxl
12 0x60000 0x68000 .Non-Secure.rwxl
13 0x68000 0x70000 .Non-Secure.rwxl
14 0x70000 0x78000 .Non-Secure.rwxl
15 0x78000 0x80000 .Non-Secure.rwxl
16 0x80000 0x88000 .Non-Secure.rwxl
17 0x88000 0x90000 .Non-Secure.rwxl
18 0x90000 0x98000 .Non-Secure.rwxl
19 0x98000 0xa0000 .Non-Secure.rwxl
20 0xa0000 0xa8000 .Non-Secure.rwxl
21 0xa8000 0xb0000 .Non-Secure.rwxl
22 0xb0000 0xb8000 .Non-Secure.rwxl
23 0xb8000 0xc0000 .Non-Secure.rwxl
24 0xc0000 0xc8000 .Non-Secure.rwxl
25 0xc8000 0xd0000 .Non-Secure.rwxl
26 0xd0000 0xd8000 .Non-Secure.rwxl
27 0xd8000 0xe0000 .Non-Secure.rwxl
28 0xe0000 0xe8000 .Non-Secure.rwxl
29 0xe8000 0xf0000 .Non-Secure.rwxl
30 0xf0000 0xf8000 .Non-Secure.rwxl
31 0xf8000 0x100000 .Non-Secure.rwxl
Non-secure callable region 0 placed in flash region 0 with size 32.


SRAM region..Domain..Permissions
00 0x00000 0x02000.Secure..rwxl
01 0x02000 0x04000.Secure..rwxl
02 0x04000 0x06000.Secure..rwxl
03 0x06000 0x08000.Secure..rwxl
04 0x08000 0x0a000.Secure..rwxl
05 0x0a000 0x0c000.Secure..rwxl
06 0x0c000 0x0e000.Secure..rwxl
07 0x0e000 0x10000.Secure..rwxl
08 0x10000 0x12000.Non-Secure.rwxl
09 0x12000 0x14000.Non-Secure.rwxl
10 0x14000 0x16000.Non-Secure.rwxl
11 0x16000 0x18000.Non-Secure.rwxl
12 0x18000 0x1a000.Non-Secure.rwxl
13 0x1a000 0x1c000.Non-Secure.rwxl
14 0x1c000 0x1e000.Non-Secure.rwxl
15 0x1e000 0x20000.Non-Secure.rwxl
16 0x20000 0x22000.Non-Secure.rwxl
17 0x22000 0x24000.Non-Secure.rwxl
18 0x24000 0x26000.Non-Secure.rwxl
19 0x26000 0x28000.Non-Secure.rwxl
20 0x28000 0x2a000.Non-Secure.rwxl
21 0x2a000 0x2c000.Non-Secure.rwxl
22 0x2c000 0x2e000.Non-Secure.rwxl
23 0x2e000 0x30000.Non-Secure.rwxl
24 0x30000 0x32000.Non-Secure.rwxl
25 0x32000 0x34000.Non-Secure.rwxl
26 0x34000 0x36000.Non-Secure.rwxl
27 0x36000 0x38000.Non-Secure.rwxl
28 0x38000 0x3a000.Non-Secure.rwxl
29 0x3a000 0x3c000.Non-Secure.rwxl
30 0x3c000 0x3e000.Non-Secure.rwxl
31 0x3e000 0x40000.Non-Secure.rwxl

Peripheral..Domain..Status
00 NRF_P0               Non-Secure.OK
01 NRF_CLOCK            Non-Secure.OK
02 NRF_RTC1             Non-Secure.OK
03 NRF_NVMC             Non-Secure.OK
04 NRF_UARTE1           Non-Secure.OK
05 NRF_UARTE2           Secure..SKIP
06 NRF_TWIM2            Non-Secure.OK
07 NRF_SPIM3            Non-Secure.OK
08 NRF_TIMER0           Non-Secure.OK
09 NRF_TIMER1           Non-Secure.OK
10 NRF_TIMER2           Non-Secure.OK
11 NRF_SAADC            Non-Secure.OK
12 NRF_PWM0             Non-Secure.OK
13 NRF_PWM1             Non-Secure.OK
14 NRF_PWM2             Non-Secure.OK
15 NRF_PWM3             Non-Secure.OK
16 NRF_WDT              Secure..SKIP
17 NRF_IPC              Non-Secure.OK
18 NRF_VMC              Non-Secure.OK
19 NRF_FPU              Non-Secure.OK
20 NRF_EGU1             Non-Secure.OK
21 NRF_EGU2             Non-Secure.OK
22 NRF_DPPIC            Non-Secure.OK
23 NRF_GPIOTE1          Non-Secure.OK
24 NRF_REGULATORS       Secure..SKIP

SPM: NS image at 0xc000
SPM: NS MSP at 0x20025830
SPM: NS reset vector at 0xdbd5
SPM: prepare to jump to Non-Secure image.
***** Booting Zephyr OS build v2.0.99-ncs1 *****
```

**Comment by gordonklaus** [comment]
The hello_world sample is configured with `CONFIG_LTE_AUTO_INIT_AND_CONNECT=y`, which means it will automatically initialize the LTE Link Control subsystem and try to connect to the network at system initialization time (right after boot).  This can take up to a couple minutes if you have bad coverage or even fail in the worst case.  Do you have a SIM card installed?

**Comment by gordonklaus** [comment]
I suspect this issue is fixed with 8a2b6db04d68af52bf05ba5b8e4e6699daa85f5f.

Please re-open if you still have problems.

**Comment by cwbrandsdal** [comment]
Works now!
